tests/buckets_sample.rs

```diff
@@ -48,41 +53,3 @@ fn import_sample_invalid_sample() {
         assert!(json["status"].to_string().contains("Invalid sample bucket"));
     });
 }
-
-#[test]
-// Skipping for Capella as there is a bug where re-loading already loaded bucket returns 201
```

Comment:
Please add back this comment in the new place, to keep it clear why the capella variant of the test is being ignored